### PR TITLE
feat : Added Thriller Genre quizzes

### PR DESCRIPTION
**Pull request by Naman-Saxena1**
Capture1
Capture2

### CODE REVIEW

**Comment by swstica** [comment]
Clean UI and nice questions! :D
Looks good to me, can be merged.

File: Pages/Genre-Thriller-Quizzes/you-quiz.html [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+    <link href='https://fonts.googleapis.com/css?family=Comfortaa' rel='stylesheet'>
+    
+    <link rel="icon" type="image/x-icon" href="../../Images/favicon-icon.png">
+    <title>Quiztron | You</title>
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="stylesheet" href="../../topnavbar.css">
+    <link rel="stylesheet" href="../quizzes-page.css">
+</head>
+<body>
+    <div class="top-bar">
```
Comment (mehulsatardekar) [inline Pages/Genre-Thriller-Quizzes/you-quiz.html:19]: Suggestion use semantic HTML, here you could have the main tag

**Review by mehulsatardekar — APPROVED** [review]
Apart from Semantic Html, the code is nicely written. can be merged after the changes

**Review by parvez-rashid — APPROVED** [review]
The app and code looks great! Can be merged.

**Pull request merged**